requirements.txt

```diff
@@ -16,6 +18,7 @@ konfig==1.1
 Mako==1.0.9
 MarkupSafe==1.1.1
 mozsvc==0.10
+mysqlclient==1.4.6
```

Comment:
no, but I was unable to open `mysql://` DSNs using ust PyMySql alone. Adding the mysqlclient resolved that problem and allowed me to use `mysql+mysqldb://` DSNs. Since a number of stand-alone folks use mysql or maria, I figured adding that library would be a useful thing for them too.

Reply:
>  I was unable to open mysql:// DSNs using ust PyMySql alone

IIRC you need to use "pymysql://" or "mysql+pymysql://" in order to use the PyMySQL backend, because it ships as a separate package rather than being built in to sqlachemy.